main.cpp

```diff
@@ -50,89 +50,116 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
-
 //  1)
-//  Noun:
-//  action 1:
+//  Noun:   camera
+
+//  action 1:   the camera advances film
+camera.advanceFilm();
 
-//  action 2:
+//  action 2:   the camera takes a photo
+camera.takePhoto();
 
-//  action 3:
+//  action 3:   the camera rewinds film
+camera.rewindFilm();
  
 //  2)
-//  Noun:
-//  action 1:
+//  Noun:   radio       
+//  action 1:   the radio powers on
+radio.turnOn();
 
-//  action 2:
+//  action 2:   the radio tunes a station
+redio.tuneStation();
+
+//  action 3:   the radio turns up the volume
+radio.raiseVolume();
 
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
+//  Noun:   dog
+//  action 1:   the dog barks
+dog.bark();
 
-//  action 2:
+//  action 2:   the dog walks
+dog.walk();
+
+//  action 3:   the dog heels
+dog.heel();
 
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
+//  Noun:   car
+//  action 1:   the car drives
+car.drive();
 
-//  action 2:
+//  action 2:   the car brakes
+car.brake();
+
+//  action 3:   the car signals a turn
+car.signalTurn();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.